tests/conftest.py

```diff
@@ -391,3 +395,50 @@ def lang(app, lang_type):
         "type": "languages"
     })
     return lang
+
+
+@pytest.fixture(scope="module")
+def resource_type_type(app):
+    """Resource type vocabulary type."""
+    return vocabulary_service.create_type(
+        system_identity, "resource_types", "rsrct")
+
+
+@pytest.fixture(scope="module")
+def resource_type_item(app, resource_type_type):
+    """Resource type vocabulary record."""
+    return vocabulary_service.create(system_identity, {
+        "id": "image-photo",
+        "props": {
+            "csl": "graphic",
+            "datacite_general": "Image",
+            "datacite_type": "Photo",
+            "openaire_resourceType": "25",
+            "openaire_type": "dataset",
+            "schema.org": "https://schema.org/Photograph",
+            "subtype": "image-photo",
+            "subtype_name": "Photo",
+            "type": "image",
+            "type_icon": "chart bar outline",
+            "type_name": "Image",
+        },
+        "title": {
+            "en": "Photo"
+        },
+        "type": "resource_types"
+    })
+
+
+RunningApp = namedtuple("RunningApp", [
+    "app", "location", "resource_type_item"
+])
+
+
+@pytest.fixture
+def running_app(app, location, resource_type_item):
```

Comment:
Right on the money: it is needed for all(most) tests because resource type is required when creating an RDMRecord. The passed resource_type value must be validated against one in the database.

I tabled the addition of `db` / `es_clear` because they can still be added per test and there is a discussion around tests/fixtures that might be worth having: `app` and `es_clear` are the heaviest fixtures, but app is module scoped at least. If we take the time to experiment/assemble them right, we could get some nice performance improvements in tests while still keeping them isolated. In short, by not doing es_clear all the time right now, we gain some improvements, but we may be able to create a more targeted fixture for when we do need it.